.github/workflows/feature-request-closer.yml

```diff
@@ -0,0 +1,23 @@
+name: Feature Request Closer
+on:
+  schedule:
+    - cron: 40 11 * * * # Run at 11:40 AM UTC (3:40 AM PST, 4:40 AM PDT)
+  workflow_dispatch:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+      - name: Install Actions
+        run: cd ./.github/actions && npm install --production && cd ../..
+      - name: Stale Closer
+        uses: ./.github/actions/StaleCloser
+        with:
+          labels: Feature Request
+          ignoredLabels: debugger
+          addLabels: more votes needed
+          closeDays: 14
```

Comment:
14 days seems short to close feature requests.  I was thinking we'd keep it at 60.